### ISSUE TITLE
Squares sometimes need borders?

### ISSUE THREAD

**Issue by TodePond** [TodePond/CellPond#227]
Or maybe not... I'm undecided
![image](https://user-images.githubusercontent.com/15892272/190429674-fea55993-2a07-406a-9acf-4494f39624f1.png)

**Comment by TeraByte-Official** [comment]
Just give borders to cells that are adjacent to cells with the same color.

**Comment by TodePond** [comment]
Yes that would be a good idea. It's also currently bad for grey squares:
![image](https://user-images.githubusercontent.com/15892272/191488473-494a30b1-fa12-4599-89e0-dcdbbfd08f86.png)

**Comment by TeraByte-Official** [comment]
In that case, give it a border regardless.
